src/main/java/htsjdk/samtools/cram/structure/Slice.java

```diff
@@ -330,22 +337,47 @@ public CramRecordReader createCramRecordReader(final CompressionHeader header,
 
     /**
      * Generate a CRAI Index entry from this Slice
-     * @return a new CRAI Index Entry
+     * @return a new CRAI Index Entry derived from this Slice
      */
     public CRAIEntry getCRAIEntry() {
         return new CRAIEntry(sequenceId, alignmentStart, alignmentSpan, containerOffset, offset, size);
     }
+
     /**
-     * Generate a CRAI Index entry from this Slice and the container offset.
+     * Generate a CRAI Index entry from this Slice and other container parameters,
+     * splitting Multiple Reference slices into constituent reference sequence entries.
      *
-     * TODO: investigate why we sometimes need to pass in an external containerStartOffset
+     * TODO: investigate why we sometimes need to pass in an external containerStartByteOffset
      * because this Slice's containerOffset is incorrect
      *
-     * @param containerStartOffset the byte offset of this Slice's Container
-     * @return a new CRAI Index Entry
+     * TODO: investigate why we sometimes split multi-ref and sometimes do not
```

Comment:
i.e. why do we need both this method and the one above it?

this replaces `CRAIIndex.processContainer()`